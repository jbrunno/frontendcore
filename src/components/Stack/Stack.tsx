import React from 'react';
import { Stack as MuiStack, StackProps as MuiStackProps } from '@mui/material';

export type StackProps = MuiStackProps;

export function Stack(props: StackProps) {
  return <MuiStack {...props} />;
}

export default Stack;
