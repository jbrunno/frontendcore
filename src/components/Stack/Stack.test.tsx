import React from 'react';
import { render } from '@test-utils/component-testing';
import { Stack } from './Stack';

test('renders the Stack', () => {
  const { container } = render(<Stack />);
  expect(container).toMatchSnapshot();
});
