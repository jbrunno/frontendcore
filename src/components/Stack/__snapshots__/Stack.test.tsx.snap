// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Stack 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div>
  <div
    class="emotion-0"
  />
</div>
`;
